code/fmriprep/ss-fmriprep.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+#SBATCH --partition=russpold
+#SBATCH --mem=55GB
+#SBATCH --cpus-per-task=16
+#SBATCH --time=20:00:00
+#SBATCH --job-name=fmriprep
+#SBATCH --error="slurm-%A_%a.err"
+
+# Submit a Single Session through fMRIPrep
+
+ARGS=($@)
+DATADIR=$1
+STUDY=`basename $DATADIR`
+
+if [[ -n $SLURM_ARRAY_TASK_ID ]]; then
+  SES=${ARGS[`expr ${SLURM_ARRAY_TASK_ID} + 1`]}
+else
+  SES=$2
+fi
+
+echo "Processing: $SES"
+
+IMG="/oak/stanford/groups/russpold/users/cprovins/singularity_images/fmriprep-23.1.4.simg"
+
+WORKDIR="${L_SCRATCH}/fmriprep/${STUDY}/${SES}"
+mkdir -p ${WORKDIR}
+OUTDIR="${DATADIR}/derivatives"
+mkdir -p $OUTDIR
+
+PATCHES=""
+
+BINDINGS="-B $DATADIR/inputs:/data:ro \
+-B ${WORKDIR}:/work \
+-B ${OUTDIR}:/out \
+-B $DATADIR/code/license.txt:/opt/freesurfer/license.txt \
+-B $DATADIR/code/filter_file_$SES.json:/filter_file_$SES.json \
```

Comment:
this is going to crowd the code directory with small json files. I suggest two actions:

1. make this file be created under /tmp (or some other temporary file resource that is cleaned up automatically)
2. send an issue to fmriprep to request that bids filters are reported in the log file very clearly (and perhaps make them also appear in the config).

Reply:
Both are done